source/common/grpc/async_client_impl.h

```diff
@@ -49,9 +49,10 @@ class AsyncClientImpl final : public AsyncClient<RequestType, ResponseType> {
 
   AsyncStream<RequestType>* start(const Protobuf::MethodDescriptor& service_method,
                                   AsyncStreamCallbacks<ResponseType>& callbacks) override {
+    const Optional<std::chrono::milliseconds> no_timeout;
```

Comment:
I see, it gets used in the context of initialize(). Sure open to suggestions on naming. Same problem applies to pointers or references. Not sure if there is a good naming scheme.